_episodes/03-technical.md

```diff
@@ -101,6 +101,38 @@ are modifying.
 > screenshared.
 {: .challenge}
 
+## Configuring and Sync Forks
+
+Forks are often used in open source development on GitHub 
+to make changes to a project without affecting the original repository. 
+You must first configure a remote that points to the upstream repository in Git.
+Then, you can sync and fetch updates from or 
+submit changes to the original repository with pull requests. 
+
+To **configure a remote** for a fork, follow these commands:
+
+~~~
+$ git remote -v
+$ git remote add upstream https://github.com/ORIGINAL_OWNER/ORIGINAL_REPOSITORY.git
```

Comment:
$ git remote add original https://github.com/ORIGINAL_OWNER/ORIGINAL_REPOSITORY.git

- Be consistent with the language

Reply:
If that's the goal, let's please keep consistent with GitHub language: `upstream` instead of `original`.

Reply:
I also oppose original. Your remotes will be `origin` and `original`? Not considering what it does to tab-completion, this is very confusing.

Reply:
+1 for upstream